__tests__/app.test.js

```diff
@@ -275,3 +275,54 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("202: updates the current vote by incrementing the value provided in the request", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: -179 })
+      .expect(202)
```

Comment:
We should expect a 200 for a successful patch 👍 202 indicates that the processing may not yet be completed.